### ISSUE TITLE
[Feature request] Dark mode for the editor

### ISSUE THREAD

**Issue by replaceitem** [Calsign/APDE#100]
I'd love a dark theme for the code section (dark background, white text, maybe also adjusted syntax highlighting colors if it's not readable).

P.s.
I really love the app, i can code everywhere i want, thank you so much!!!

**Comment by Erdien** [comment]
sometimes (espetialy at night) I need dark mode to work, Ind I'm 80% on your side!!
I'm not sure, but I think, that by tradition processing's theme is light, with black/blue details so why you should break traditions?
On the other hand this is mobile independent? app, so it would be nice to not look at flashlight whole time.

**Comment by replaceitem** [comment]
Yeah i agree, processing doesnt have a theme setting, but it has the theme.txt file, where every syntax color and background color can be edited, so i would suggest for APDE that a theme.txt file in the sketchbook could be a good solution.
